parlai/core/agents.py

```diff
@@ -149,9 +149,7 @@ def reset_metrics(self):
         self.metrics.clear()
 
     def share(self):
-        """If applicable, share any parameters needed to create a shared version
```

Comment:
just changed it to a more accurate description, this comment was copypasta from Agents.share()